test/views/OptOutReturnDeadlinesViewSpec.scala

```diff
@@ -48,7 +48,7 @@ class OptOutReturnDeadlinesViewSpec extends ViewBaseSpec {
 
     val backLink = ".govuk-back-link"
 
-    val overdueLabel = ".task-overdue"
+    val overdueLabel = ".govuk-tag--red"
```

Comment:
Nothing to do with your change but could we add an additional test to the scenarios where a return is not overdue that checks the tag isn't present.